dashboard/src/components/Catalog/Catalog.tsx

```diff
@@ -250,7 +266,7 @@ export default function Catalog() {
     .filter(
       c =>
         filters[filterNames.CATEGORY].length === 0 ||
-        intersection(filters[filterNames.CATEGORY], getOperatorCategories(c)).length,
+        intersection(filters[filterNames.CATEGORY], getOperatorCategories(c)).length > 0,
```

Comment:
Out of curiosity: any special reason for this change? wouldn't just `...X.length` check every falsy value, which subsumes the length===0 case ?

Reply:
Other than a tendency to prefer explicit over implicit, I don't remember. Vague memory of a test failing and having to update code, but can't see how that could have affected this line.

Reply:
Also reverting this change, since I need to update the branch anyway.